plugins/generator-1.18.2/forge-1.18.2/procedures/entity_registry_name.java.ftl

```diff
@@ -0,0 +1 @@
+(${input$entity} != null ? ForgeRegistries.ENTITIES.getKey(${input$entity}.getType()).toString() : ForgeRegistries.ENTITIES.getDefaultKey().toString())
```

Comment:
```suggestion
((${input$entity} != null ? ForgeRegistries.ENTITIES.getKey(${input$entity}.getType()) : ForgeRegistries.ENTITIES.getDefaultKey()).toString())
```

This should be applied to 1.19 as well

Reply:
Huh, I thought I had done that, in that case I think I overrode the 1.18 version with the 1.19 version, because one of the two should say "ENTITY_TYPES"

Reply:
Done! Added it to the 1.19 version with the correct naming